gatsby-node.js

```diff
@@ -227,12 +227,21 @@ exports.onCreatePage = ({ page, actions }) => {
   // Recreates the page with the messages that ReactIntl needs
   // This will be passed to the ReactIntlProvider Component
   // Used within gatsby-browser.js and gatsby-ssr.js
+  const context = { ...page.context };
+  const locale = context.locale || nodeLocales.defaultLanguage;
+  const isLearnPage = context.categoryName === 'learn';
+  if (isLearnPage) {
```

Comment:
I don't think you need to make a constant here.. Just `if (context.categoryName === 'learn')`